content/en/blog/_posts/2022-12-12-host-process-containers-stable/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Windows HostProcess Containers Are Going Stable"
+date: 2022-12-13
+slug: host-process-containers-stable
+---
+
+**Authors**: Mark Rossetti (Microsoft) and Brandon Smith (Microsoft)
+
+The long-awaited day has arrived: HostProcess containers, the Windows equivalent to Linux privileged
+containers, has finally made it to **GA in Kubernetes 1.26**!
+
+Let's recap - what are HostProcess containers and why are they useful? Cluster operators
+are often faced with the need to configure their nodes upon provisioning. Whether it's
+installing services, certificates, network configs, device plugins, or even monitoring solutions like
+prometheus, HostProcess containers enable you to it all with minimal effort.
```

Comment:
```suggestion
installing Windows services, configuring keys and certificates for TLS, managing network configurations, providing a [device plugin](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/),
or even deploying monitoring tools such as a Prometheus exporter, HostProcess containers
enable you to do it all with minimal effort.
```

Reply:
I will update this to say `configuring registry keys` once i commit the original suggestion.